src/plugins/mer/mergeneraloptionswidget.ui

```diff
@@ -137,6 +137,16 @@
         </property>
        </widget>
       </item>
+      <item>
+       <widget class="QCheckBox" name="importQmakeVariablesCheckBox">
+        <property name="toolTip">
+         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Track changes in project *.spec and *.yaml files and import variables from these files to qmake step.&lt;/p&gt;&lt;p&gt;You are able to define variables in *.yaml file in QMakeOptions section or in *.spec file in qtc_qmake5 section. If you enable import, such variables may be referred in *.pro file using&lt;br&gt;$$VAR_NAME syntax.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+        </property>
+        <property name="text">
+         <string>Import variables from *.spec (*.yaml) file to qmake step</string>
```

Comment:
Quoted for convenience:

> Import variables from *.spec (*.yaml) file to qmake step
> 
> Track changes in project *.spec and *.yaml files and import variables from these files to qmake step.
> 
> You are able to define variables in *.yaml file in QMakeOptions section or in *.spec file in qtc_qmake5 section. If you enable import, such variables may be referred in *.pro file using $$VAR_NAME syntax.

This is much better, but still:

1. I would put more stress on the fact that this influences just the code model on QtC side, that it does not influence build results
2. No need to mention changes would be tracked - that's inevitable
3. There is nothing like "qtc_qmake5 section"
4. The variables may be referred in *.pro file in any case, just the code model on QtC side will not be happy with this disabled
5. It needs build engine running

I suggest the following formulation:

Label:

> Augment project model using actual qmake arguments

Tool tip:

> The effective qmake call reported in build configuration is missing further arguments possibly supplied e.g. according to the recipe in the .spec (.yaml) file. This may render the project model inaccurate.
> 
> This option enables to augment the project model with qmake variable assignments found on the actual qmake command line.
> 
> When enabled, qmake will be invoked automatically when the .spec (.yaml) file is updated, which may prompt you to start the Sailfish OS build engine if it is not running.

Reply:
Fixed. You are so right about: 

> hard to say something meaningful here.
I bet - only few people can get to the essence.:)